### PR TITLE
Allow Guzzle 7 as laravel 8 requires it

### PR DESCRIPTION
**Pull request by ziming**
Will be great if you can mark it hacktoberfest-accepted too

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "require": {
         "php": "^7.2",
         "ext-simplexml": "*",
-        "guzzlehttp/guzzle": "^6.2",
+        "guzzlehttp/guzzle": "^6.2|^7.0",
         "illuminate/notifications": "~5.8 || ~6.0 || ^7.0 || ^8.0",
         "illuminate/support": "~5.8 || ~6.0 || ^7.0 || ^8.0",
         "orchestra/testbench": "~3.8.0 || ^4.0 || ^5.0"
```
Comment (rostockahoi) [inline composer.json:20]:
```suggestion
        "orchestra/testbench": "~3.8.0 || ^4.0 || ^5.0 || ^6.0"
```

**Comment by rostockahoi** [comment]
Looks like `orchestra/testbench` should also be bumped.

**Comment by ziming** [comment]
Done! Will be great if u can mark it hacktoberfest-accepted too

**Comment by mbardelmeijer** [comment]
Cheers!

**Pull request merged**